run_mount_test.go

```diff
@@ -99,7 +99,35 @@ VOLUME /foo
 		"mountpoint", "-q", "/foo").AssertOK()
 }
 
-func TestCopyingUpInitialContentsOnVolume(t *testing.T) {
+func TestCopyingUpInitialContentsOnVolumes(t *testing.T) {
+	t.Parallel()
+	testutil.RequiresBuild(t)
+	base := testutil.NewBase(t)
+	const imageName = "nerdctl-test-copying-initial-content"
+	defer base.Cmd("rmi", imageName).Run()
+	defer base.Cmd("volume", "rm", "copying-initial-content").Run()
+
+	dockerfile := fmt.Sprintf(`FROM %s
+RUN mkdir -p /mnt && echo hi > /mnt/initial_file
+CMD ["cat", "/mnt/initial_file"]
+        `, testutil.AlpineImage)
+
+	buildCtx, err := createBuildContext(dockerfile)
+	assert.NilError(t, err)
+	defer os.RemoveAll(buildCtx)
+
+	base.Cmd("build", "-t", imageName, buildCtx).AssertOK()
+
+	//AnonymousVolume
+	base.Cmd("run", "--rm", imageName).AssertOutContains("hi")
+	base.Cmd("run", "-v", "/mnt", "--rm", imageName).AssertOutContains("hi")
+
+	//NamedVolume
+	base.Cmd("volume", "create", "copying-initial-content").AssertOK()
```

Comment:
The volume should be created automatically to fix https://github.com/containerd/nerdctl/issues/262

Can be another PR, though

Reply:
I did both @AkihiroSuda